Description: Suppress ffmpeg version mismatch error
 Requiring an exact ffmpeg version is usually not a good idea in a binary
 distribution because:
 - All FFmpeg security updates require a subsequent binNMU of mpv.
 - Debian generated dependencies do not capture this dependency well (at least
   without extra hacking).
 - The requirement itself usually indicates an ABI violation.
 For these reasons, remove the check and assume the current FFmpeg version is
 compatible.
Author: James Cowgill <jcowgill@debian.org>
Bug-Debian: https://bugs.debian.org/831537
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/player/main.c
+++ b/player/main.c
@@ -384,15 +384,6 @@ int mp_initialize(struct MPContext *mpct
     if (handle_help_options(mpctx))
         return 1; // help
 
-    if (!print_libav_versions(mp_null_log, 0)) {
-        print_libav_versions(mpctx->log, MSGL_FATAL);
-        MP_FATAL(mpctx, "\nmpv was compiled against an incompatible version of "
-                 "FFmpeg/Libav than the shared\nlibrary it is linked against. "
-                 "This is most likely a broken build and could\nresult in "
-                 "misbehavior and crashes.\n\nThis is a broken build.\n");
-        return -1;
-    }
-
     if (!mpctx->playlist->first && !opts->player_idle_mode) {
         // nothing to play
         mp_print_version(mpctx->log, true);
